spec/karma.conf.js

```diff
@@ -32,18 +32,10 @@ module.exports = (config) => {
     exclude: [
     ],
 
-
-    // preprocess matching files before serving them to the browser
-    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
-    preprocessors: {
-      'javascripts/main.js': ['coverage']
-    },
-
-
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ['spec', 'coverage'],
+    reporters: ['spec'],
```

Comment:
as bundle size grows this became way to slow and also failed couple times with heap overflow 😮. I removed it for now, I'd say coverage is only nice to have